migration/instructions-for-infra-team.md

```diff
@@ -77,17 +77,19 @@ JiraKey,GitHubUrl,GitHubNumber
 LUCENE-1,https://github.com/apache/lucene/issues/1080,1080
 ```
 
+Peek at the issue through GitHub and confirm it was imported and looks reasonable.
+
 ### Clean up the test data
 
-Once the test is done, please delete `mapping-data/issue-map.csv` file and the imported issue (only admin accounts can delete an issue) before the actual migration.
+Once the test is done, please delete `mapping-data/issue-map.csv` local file and the imported issue through GitHub (only admin accounts can delete an issue) before the actual migration.
 
 ## Run the import script
 
-Please specify the `min` option to 1 and `max` option to the maximum number of the Lucene Jira issue, that will be known by then.
+Please specify the `min` option to 1 and `max` option to the maximum number of the Lucene Jira issue, that will be known by then (how?  based on the un-tgz'd GitHub import data?  or maybe you/we will communicate this number to them?  if so, let's say that above?)
```

Comment:
Here I thought we should explicitly state that we will tell them the max id?

Reply:
Thanks for reviewing it.
This is the only part we cannot determine yet, I will replace the `<will be known>` with the concrete number after taking the Jira snapshot to avoid miscommunication.

Reply:
Oh I see, `<will be known>` means "we will fill this in before we hand off these instructions to INFRA on the actual day of migration", OK.  I'll try to word that in.

Reply:
Thank you very much!